src/main/resources/db/migration/all/20220929141644__alter_premises_table_to_add_extra_columns.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE premises ADD COLUMN service TEXT;
```

Comment:
I'm assuming this is to discriminate between CAS1 and CAS3 premises?  If so presume this is going to be required on every Premises moving forwards.  So maybe something like:

```
ALTER TABLE premises ADD COLUMN service TEXT NOT NULL DEFAULT 'CAS1';
```